src/sentry/static/sentry/app/data/experimentConfig.tsx

```diff
@@ -0,0 +1,45 @@
+import {Experiments} from 'app/types/experiments';
+
+/**
+ * This is the value an experiment will have when the unit of assignment
+ * (organization, user, etc) is not part of any experiment group.
+ *
+ * This likely indicates they should see nothing, or the original version of
+ * what's being tested.
+ */
+export const unassignedValue = -1;
+
+/**
+ * Frontend experiment configuration object
+ */
+export const experimentList = [
```

Comment:
Should this be typed as `ExperimentConfig[]`?

Reply:
So this is kind of annoying. I want this to be a `as const` type, so I can do all the type inference to get explicit type values out of this. But if I mark it as `ExperimentConfig[]` and get type safety on the const itself, it no longer has const values, and everything widens up to just the general types, so instead of getting a nice list of possible experiment keys, you just get `string`.

I wish there was some way to type this as const, but enforce the declaration to extend Experiment config. Maybe there's some way to do this?